packages/frontend/components/primitives/dashboard-link/styles.ts

```diff
@@ -22,8 +22,8 @@ export const icon = ({ spaces }: Theme) => css`
 
 export const img = () => css`
   display: block;
-  width: 22px;
-  height: 22px;
+  width: 18px;
+  height: 18px;
```

Comment:
dashboard-link component is not a primitive(not used across the App) + use button for it